### ISSUE TITLE
Documentation Error in "A Miscellany of Leo Scripting"

### ISSUE THREAD

**Issue by tbpassin** [leo-editor/leo-editor#826]
In the section headed "Modifying the body pane directly", it is stated that there is a method:

```
   w.replace(i,j,s)                    # Replace the text from i to j by s.
```
I find that this method does not exist (at least for 64-bit Python 2.7, Qt4, Windows).

Steps to reproduce
============
1.  Create a node with headline "@command xx-test-replace"
2. Enter this code in the node body:
```
    w = c.frame.body.wrapper # Leo's body pane
    g.es(w.insert)
    g.es(w.replace)
```
3. Save the Leo outline and reload to make the new command available.
4. Activate the xx-test-replace using Alt-x and the minibuffer.
5. Observe that the log pane shows a bound method for w.insert.
6. Observe that the log pane shows an exception for w.replace:
```
    exception executing script
    AttributeError: 'QTextEditWrapper' object has no attribute 'replace'
```
Also, running help() on w does not list a `replace` method.

**Comment by edreamleo** [comment]
Fixed at 0504a4e and on the web.

**Comment by edreamleo** [comment]
On Sun, Mar 18, 2018 at 9:52 PM, tbpassin <notifications@github.com> wrote:

> In the section headed "Modifying the body pane directly", it is stated
> that there is a method:
>
> w.replace(i,j,s) # Replace the text from i to j by s.
>
> I find that this method does not exist (at least for Python 2.7, Qt4).
>
​Good catch.  I have just fixed it.

Edward
